plugins/woocommerce-admin/client/two-column-tasks/task-list-completion-slot/task-list-completion-slot.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { useSlot } from '@woocommerce/experimental';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import {
+	WC_TASK_LIST_COMPLETION_SLOT_NAME,
+	WooTaskListCompletionItem,
+} from './utils';
+
+export const TaskListCompletionSlot = ( {
```

Comment:
It seems that we need to add these similar codes (`*slot.tsx`+`utils.tsx`) for every Extensibility PRs.  Wondering if it's possible and a good idea (?) to create a helpful function to build a slotfill component with ease? 😬

Reply:
Good thinking, @chihsuan! I wonder if there's prior art somewhere, considering Slotfill is widely used?

Reply:
Might be, I reused much inspiration from the footer slotfill! I was thinking to leave the de-duplication until there's a clearer idea of what needs we have for the slotfills before creating an abstraction layer on top.

I was also thinking the *utils.tsx would be moved over to the @woo/extends package, maybe